specification/postgresqlhsc/resource-manager/Microsoft.DBforPostgreSQL/stable/2022-11-08/postgresqlhsc.json

```diff
@@ -0,0 +1,2987 @@
+      "type": "object",
+      "properties": {
+        "message": {
+          "type": "string",
+          "description": "Error Message."
+        },
+        "nameAvailable": {
+          "type": "boolean",
+          "description": "Indicates whether the resource name is available."
+        },
+        "name": {
+          "type": "string",
+          "description": "name of the cluster."
+        },
+        "type": {
+          "type": "string",
+          "description": "type of the cluster"
+        }
+      }
+    },
+    "PrivateEndpointProperty": {
+      "type": "object",
+      "description": "Property to represent resource id of the private endpoint.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource id of the private endpoint."
+        }
+      }
+    },
+    "PrivateEndpointConnection": {
+      "description": "A private endpoint connection",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v4/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateEndpointConnectionSimpleProperties",
+          "x-ms-client-flatten": true,
+          "description": "Properties of the private endpoint connection."
+        }
+      }
+    },
+    "PrivateEndpointConnectionProperty": {
+      "description": "A private endpoint connection property",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v4/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateEndpointConnectionProperties",
+          "x-ms-client-flatten": true,
+          "description": "Properties of the private endpoint connection."
+        }
+      }
+    },
+    "PrivateEndpointConnectionActionRequest": {
+      "description": "A private endpoint connection",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateEndpointConnectionActionRequestProperties",
+          "x-ms-client-flatten": true,
+          "description": "Properties of the private endpoint connection."
+        }
+      },
+      "required": [
+        "properties"
+      ]
+    },
+    "PrivateEndpointConnectionListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnectionProperty"
+          },
+          "description": "The list of private endpoint connections"
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "description": "A list of private endpoint connections."
+    },
+    "PrivateLinkResource": {
```

Comment:
>PrivateLinkResource [](http://example.com/codeflow?start=5&length=19)

use the definitions from here instead : 

[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v4/privatelinks.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v4/privatelinks.json)

Reply:
Done